src/plasma/plasma_store.cc

```diff
@@ -213,10 +238,9 @@ int create_object(Client *client_context,
   get_malloc_mapinfo(pointer, &fd, &map_size, &offset);
   assert(fd != -1);
 
-  entry = (object_table_entry *) malloc(sizeof(object_table_entry));
-  memset(entry, 0, sizeof(object_table_entry));
-  memcpy(&entry->object_id, &obj_id, sizeof(entry->object_id));
-  entry->info.obj_id = obj_id;
+  entry = new object_table_entry();
```

Comment:
Ok, we can call `new` because we're sticking entry in a hash table which will never try to move or copy or delete the entry. If we were sticking it in a `utarray` then we would have problems when the array gets resized, right?